open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataElementActionUtilsImpl.java

```diff
@@ -84,57 +87,39 @@ public boolean gotoSelectedFeature(long deId, boolean flyTo)
     @Override
     public synchronized boolean gotoSelectedFeature(long[] dataElementIds, boolean flyTo)
     {
-        GeographicBoundingBox gbb = null;
-        Projection proj = getToolbox().getMapManager().getProjection(Viewer3D.class).getSnapshot();
+        GeographicBoundingBox bbox = null;
+        Projection proj = getToolbox().getMapManager().getProjection().getSnapshot();
         for (long deId : dataElementIds)
         {
             MapGeometrySupport mgs = MantleToolboxUtils.getDataElementLookupUtils(myToolbox).getMapGeometrySupport(deId);
             if (mgs != null)
             {
-                if (gbb == null)
+                if (bbox == null)
                 {
-                    gbb = mgs.getBoundingBox(proj);
+                    bbox = mgs.getBoundingBox(proj);
                 }
                 else
                 {
-                    gbb = GeographicBoundingBox.merge(gbb, mgs.getBoundingBox(proj));
+                    bbox = GeographicBoundingBox.merge(bbox, mgs.getBoundingBox(proj));
                 }
             }
             else
             {
                 return false;
             }
         }
-
-        if (gbb != null)
+        if (bbox != null)
         {
             if (LOGGER.isDebugEnabled())
             {
-                LOGGER.debug((flyTo ? "FLYTO: " : "GOTO: ") + gbb.toString());
+                LOGGER.debug((flyTo ? "FLYTO: " : "GOTO: ") + bbox.toSimpleString());
             }
             DynamicViewer view = myToolbox.getMapManager().getStandardViewer();
-            ViewerAnimator animator;
-            if (gbb.getWidth() > 0.0 || gbb.getHeight() > 0.0)
-            {
-                animator = new ViewerAnimator(view, gbb.getVertices(), true);
-            }
-            else
-            {
-                animator = new ViewerAnimator(view, gbb.getCenter());
-            }
-            if (flyTo)
-            {
-                animator.start();
-            }
-            else
-            {
-                animator.snapToPosition();
-            }
+            DataTypeActionUtils.gotoBoundingBox(bbox, view,flyTo);

```

Comment:
nitpick: there should be a space between the comma and flyTo

Reply:
oh wait i fixed a different comma mistake , shoot